dandi/due.py

```diff
@@ -55,7 +55,7 @@ def _donothing_func(*args, **kwargs):
 
 
 try:
-    from duecredit import due, BibTeX, Doi, Url, Text
+    from duecredit import due, Doi  # lgtm [py/unused-import]
```

Comment:
Let's skip this file as well from lgtm checks - it is a stub which comes from duecredit and better be left unmodified

Reply:
Do you want me to revert the changes to our local `due.py`?

Reply:
yes, to due.py and `_version.py` and list them to be excluded in to be added lgtm.yml per https://lgtm.com/help/lgtm/customizing-file-classification

Reply:
oh I see you added already lgtm.yml -- great

Reply:
Done.